mycroft/client/text/text_client.py

```diff
@@ -171,6 +172,25 @@ def load_settings():
     global max_log_lines
     global show_meter
 
+    config_file = None
+
+    # Old location
+    path = os.path.join(os.path.expanduser("~"), ".mycroft_cli.conf")
+    if os.path.isfile(path):
+        config_file = path
```

Comment:
can we `shutil.move` the file to the xdg directory automatically instead?

Reply:
I think as this is an user-exposed config file (similar to the config files) it shouldn't be moved automatically until the location has been deprecated for a while.

Reply:
add a deprecation warning like in config then?